apps/damap-frontend/src/app/services/config.service.ts

```diff
@@ -43,34 +44,103 @@ export class ConfigService {
             console.warn('App title is missing in the config');
           }
           this.configSubject.next(config);
-          const authConfig: AuthConfig = {
-            issuer: config.authUrl,
-            clientId: config.authClient,
-            redirectUri: window.location.origin,
-            logoutUrl: window.location.origin,
-            oidc: true,
-            scope: config.authScope,
-            // useSilentRefresh: true,
-            responseType: 'code',
-            showDebugInformation: isDevMode(),
-            // sessionChecksEnabled: true,
-          };
-          this.oauthService.configure(authConfig);
-          this.oauthService.setupAutomaticSilentRefresh();
-          return this.oauthService
-            .loadDiscoveryDocumentAndTryLogin()
-            .then(() => {
+
+          if (config.env === 'PROD') {
```

Comment:
Here the SPA sends a dummySecret with useHttpBasicAuth. A browser client can not keep secrets. For eduID/public clients this should be PKCE-only. Can we drop the secret/basic auth here and confirm backend #411 works with pure PKCE?